www/nextcloud-client/Portfile

```diff
@@ -35,18 +32,22 @@ compiler.library_path
 compiler.blacklist-append {clang < 602} macports-clang-3.4 macports-clang-3.3
 compiler.fallback-append macports-clang-5.0
 
-depends_lib-append      port:qtkeychain
+depends_lib-append      port:openssl11 port:sqlite3 port:zlib port:qtkeychain
 
-depends_build-append    bin:git:git
+depends_build-append    bin:git:git \
+                        bin:inkscape:inkscape
```

Comment:
why is `inkscape` now a build dependency?

Reply:
And if it is, why would we want to allow a copy of inkscape that is installed outside of MacPorts to satisfy the dependency? (To prevent that, write `path:bin/inkscape:inkscape` instead.)

Reply:
`zlib` should also be listed as a dependency here (likely under `depends_lib`

Reply:
I added inkscape as a dependency because the build process uses it to generate bitmaps from the included SVG icons.

I am not a really a big fan of having nextcloud-client depend on inkscape just to generate 15 PNGs out of 2 SVGs.  
As an alternative we might want to pre-generate the PNGs and then "inject" them into the build folder.
That would require us to (in the worst case) update the PNGs for every update of the Portfile.

@reneeotten What do you mean? zlib is already a `depends_lib`.